ci/generate_lerna_config.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+
+PACKAGES_TOTAL=($(cat lerna_packages))
+PACKAGES_IGNORE=(@uma/serverless-orchestration @uma/affiliates)
+CI_CONFIG_FILE=".circleci/lerna_config.yml"
+
+echo "All packages modified:"
+echo ${PACKAGES_TOTAL[*]}
+
+for IGNORE in ${PACKAGES_IGNORE[@]}
+do
+   PACKAGES_ARRAY=(${PACKAGES_TOTAL[@]/$PACKAGES_IGNORE})
+done
+
+if [ ${#PACKAGES_ARRAY[@]} -eq 0 ]; then
+
+  echo "No packages for testing."
+  circleci-agent step halt;
+
+else
+
+  echo "Packages to execute:"
+  echo ${PACKAGES_ARRAY[*]}
+
+  echo "Ignored packages:"
+  echo ${PACKAGES_IGNORE[*]}
```

Comment:
As discussed today, we don't need to ignore any packages.

Reply:
Fixed.